packages/core/src/service/services/channel.service.ts

```diff
@@ -166,6 +166,7 @@ export class ChannelService {
             .select(`channel.${inverseJunctionColumnName}`, 'channelId')
             .from(junctionTableName, 'channel')
             .where(`channel.${junctionColumnName} = :entityId`, { entityId })
+            .andWhere(`${entityType.name}.id = :entityId`, { entityId })
```

Comment:
⛏ **suggestion:** use the table name instead of the entity name

```suggestion
            .andWhere(`${metadata.tableName}.id = :entityId`, { entityId })
```